VMWare/vmware-vcenter/tests/login_failure.json

```
@@ -0,0 +1,63 @@
+{
+  "input": {
+    "message": "2025-01-29T08:49:33.509Z INFO websso[52:tomcat-http--15] [CorId=2dc2573a-2356-4d19-bbd8-35fc7f33d602] [auditlogger] {\"user\":\"jesuisuntest@NT01\",\"client\":\"1.2.3.4\",\"timestamp\":\"01/29/2025 08:49:33 GMT\",\"description\":\"User jesuisuntest@NT01@1.2.3.4 failed to log in with response code 401\",\"eventSeverity\":\"INFO\",\"type\":\"com.vmware.sso.LoginFailure\"}",
+    "sekoiaio": {
+      "intake": {
+        "dialect": "VMware vCenter",
+        "dialect_uuid": "0642b03a-9d4a-4c88-a5e2-4597e366b8c4"
+      }
+    }
+  },
+  "expected": {
+    "message": "2025-01-29T08:49:33.509Z INFO websso[52:tomcat-http--15] [CorId=2dc2573a-2356-4d19-bbd8-35fc7f33d602] [auditlogger] {\"user\":\"jesuisuntest@NT01\",\"client\":\"1.2.3.4\",\"timestamp\":\"01/29/2025 08:49:33 GMT\",\"description\":\"User jesuisuntest@NT01@1.2.3.4 failed to log in with response code 401\",\"eventSeverity\":\"INFO\",\"type\":\"com.vmware.sso.LoginFailure\"}",
+    "event": {
+      "category": [
+        "network"
+      ],
+      "outcome": "failure",
+      "reason": "User jesuisuntest@NT01@1.2.3.4 failed to log in with response code 401",
+      "type": [
+        "connection"
+      ]
```

Can we change `event.category` and `event.type` to `authentication` and `start`?
```suggestion
      "category": [
        "authentication"
      ],
      "outcome": "failure",
      "reason": "User jesuisuntest@NT01@1.2.3.4 failed to log in with response code 401",
      "type": [
        "start"
      ]
```